modules/repository-gcs/src/internalClusterTest/java/org/elasticsearch/repositories/gcs/GcsProxyIntegrationTests.java

```diff
@@ -41,7 +40,6 @@
 import static org.elasticsearch.repositories.gcs.GoogleCloudStorageRepository.CLIENT_NAME;
 
 @SuppressForbidden(reason = "We start an HTTP proxy server to test proxy support for GCS")
-@LuceneTestCase.AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/93811")
```

Comment:
Does this cover all the failing tests? I see that we have a few open issues.

Reply:
Yes, it does! All failing tests are from the parent `GcsProxyIntegrationTests` class, so removing this annotation should turn it on.